doc/main.html

```diff
@@ -501,6 +533,8 @@ <h4>Conformance section</h4>
 
         <p>This section is prohibited unless the document is a Standard or Recommended Practice.</p>
 
+        <p class="note">If <code>pubType</code> is set to <code>EG</code> or <code>ER</code>, the usage of Conformance Notation is disallowed throughout the document.</p>
```

Comment:
This PR is for ER and RDD, so I would not make changes related to EGs.

Reply:
This is not a change to EG, just documenting that, as it also applies to ER.

Reply:
I do not think it is a good idea to repeat here requirements that are stated elsewhere. The error message should be sufficient.

Reply:
Removed in https://github.com/SMPTE/html-pub/pull/315/commits/55f44909e5b0da22950138775e1f1b0b8bca39cf